docs/sources/installation/binaries.md

```diff
@@ -80,6 +80,8 @@ exhibit unexpected behaviour.
 
 ## Get the docker binary:
```

Comment:
This heading needs a tweak if you don't mind when you are in there removing the colon.